conclave-cloud-api.yml

```diff
@@ -199,6 +199,110 @@ paths:
         default:
           $ref: "#/components/responses/GenericError"
 
+  /projects/{projectUID}/storage:
+    get:
+      description: Get the files stored in a project.
```

Comment:
Perhaps it should be `List the files...`. Also maybe the path should be /files instead of storage. Then again it depends on the context.

Reply:
I think we can keep it as `/storage` because, after the implementation of directories, we will not only list the files on that path but the directories contained in it too.